arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -488,7 +488,8 @@
         },
         "location": {
           "description": "The location of the resource. This cannot be changed after the resource is created.",
-          "type": "string"
+          "type": "string",
+          "readOnly": true

```

Comment:
Resource is used during Create operation. By specifying it as readonly, will the user be able to pass in the location?